File: striver/S3L1/second_largest_element.py
size = int(input())
arr = [0]*size
for i in range(0,size):
    arr[i] = int(input())
def sle(arr):
    m1 = arr[0]
    m2 = arr[0]
    for i in range (1,size):
        if(arr[i]>m1):
            m2=m1
            m1=arr[i]
        elif (arr[i]>m2):
            m2 = arr[i]
    print("largest element is ",m1," Second largest element is ",m2)
sle(arr)

File: striver/S3L1/Second_largest_element.java
package S3L1;

import java.util.Scanner;

public class Second_largest_element {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int size = sc.nextInt();
        int[] arr = new int[size];
        for (int i = 0; i < size; i++) {
            arr[i] = sc.nextInt();
        }
        sle(arr);
    }
    public static void sle(int[] arr){
        int m1 = arr[0];
        int m2 = arr[0];
        for(int i = 1;i<arr.length;i++){
            if (arr[i] > m1 ){
                m2 = m1;
                m1 = arr[i];
            }
            else if(arr[i]>m2){
                m2 = arr[i];
            }
        }
        System.out.println("highest elemnt is "+m1+" second highest element is "+m2);
    }
}
